redaxo/src/core/pages/system.log-external.php

```diff
@@ -0,0 +1,61 @@
+<?php
+
+/**
+ * @package redaxo5
+ */
+
+$func = rex_request('func', 'string');
+$logFile = ini_get('error_log');
+/*
+$logFile = rex_request('file', 'string');
+$availableLogs = [ini_get('error_log')];
+
+if (!in_array($logFile, $availableLogs)) {
+    unset($logFile);
+}
+*/
+
+if (!isset($logFile)) {
+    return;
+}
+
+$content = '
+            <table class="table table-hover">
+                <tbody>';
+
+$file = new SplFileObject($logFile, 'r');
+$file->seek(PHP_INT_MAX);
+$last_line = $file->key();
+
+$limit = 30;
+foreach (new LimitIterator($file, max(0, $last_line - $limit), $last_line) as $logLine) {
```

Comment:
hier würde ich gerne in umgekehrter reihenfolge iterieren, wie es auch im syslog ist... ideen?

Reply:
spontan fällt mir nur der weg ein, in ein array packen und `array_reverse` ...

Reply:
danke, ergänzt